go.mod

```diff
@@ -71,12 +68,23 @@ require (
 
 require github.com/go-ldap/ldap/v3 v3.4.1
 
+require (
+	github.com/decred/dcrd/dcrec/secp256k1/v4 v4.0.0-20210816181553-5444fa50b93d // indirect
+	github.com/goccy/go-json v0.7.10 // indirect
+	github.com/lestrrat-go/backoff/v2 v2.0.8 // indirect
+	github.com/lestrrat-go/blackmagic v1.0.0 // indirect
+	github.com/lestrrat-go/httpcc v1.0.0 // indirect
+	github.com/lestrrat-go/iter v1.0.1 // indirect
+	github.com/lestrrat-go/option v1.0.0 // indirect
+)
+
 require (
 	github.com/Azure/go-ntlmssp v0.0.0-20200615164410-66371956d46c // indirect
 	github.com/go-asn1-ber/asn1-ber v1.5.1 // indirect
 	github.com/golang/protobuf v1.4.3 // indirect
 	github.com/hashicorp/errwrap v1.0.0 // indirect
 	github.com/hashicorp/go-multierror v1.1.0 // indirect
+	github.com/lestrrat-go/jwx v1.2.12-0.20211125230033-881a53637884
```

Comment:
Looks good! Happy to merge as soon as the `github.com/lestrrat-go/jwx` version is a tagged version.

Reply:
@zrhoffman released, and updated go.mod, go.sum, and vendor/modules.txt